src/elm/Page/Login.elm

```diff
@@ -294,8 +295,8 @@ focusBox id =
         |> Maybe.withDefault Cmd.none
 
 
-view : Environment -> Model -> Html Msg
-view env model =
+view : Environment -> Model -> AppInfo -> Html Msg
+view env model appInfo =
```

Comment:
Kanskje mer "korrekt" å ha appInfo før model her. Altså for å gruppere Environment og AppInfo i parameterlisten da de hører bedre sammen og det åpner opp for bedre eventuell partial application.